plugin.yml

```diff
@@ -11,16 +13,15 @@ permissions:
     elevator.*:
         default: op
         description: "Gib einem Spieler Zugriff auf alle Funktionen dieses Plugins."
```

Comment:
We can remove that parent permission. It's useless now.

Reply:
It isn't

Reply:
Why do we need it.
Support for child permissions through plugin.yml was dropped and since you could give yourself the other permissions, I don't see why to keep the parent one

Reply:
Please remove remove the parent permission